test/integration_tests/test_mat_executor.py

```diff
@@ -54,18 +54,36 @@ def tearDownClass(cls):
         execute_query_fetch_all("DROP TABLE UATRAC;")
 
     def test_should_mat_view_with_dummy(self):
-        materialized_query = """CREATE MATERIALIZED VIEW dummy_view (id, label)
+        materialized_query = """CREATE MATERIALIZED VIEW dummy_view0 (id, label)
```

Comment:
Any reason for these changes `dummy_view` -> `dummy_view0`?

Reply:
I was trying to distinguish the view names used for tests that have column names and the tests that don't have column name. It has no reason besides having a set of consistent names

Reply:
I see, we could maybe get rid of the numbers as they serve no purpose. Ideally we should support both types of queries, right?

Reply:
Agree. the numbers are a bit confusing and I'll come up with better names. And yes, this PR is already able to infer column types. What about I subsume the naming fixes to my next PR that adds more test cases?